Misc/NEWS.d/next/Library/2020-10-26-18-01-09.bpo-42151.et5f7s.rst

```diff
@@ -0,0 +1,3 @@
+Make the pure Python implementation of `xml.etree.ElementTree` behave the
```

Comment:
`[2] ../Misc/NEWS.d/next/Library/2020-10-26-18-01-09.bpo-42151.et5f7s.rst:1: default role used``

Documentation build is failed